joern-cli/build.sbt

```
@@ -129,28 +129,23 @@ generateScaladocs := {
   out
 }
 
-Universal / packageBin / mappings ++= sbt.Path.directory(new File("joern-cli/src/main/resources/scripts"))
-
-lazy val removeModuleInfoFromJars = taskKey[Unit]("remove module-info.class from dependency jars - a hacky workaround for a scala3 compiler bug https://github.com/scala/scala3/issues/20421")
-removeModuleInfoFromJars := {
-  import java.nio.file.{Files, FileSystems}
-  val logger = streams.value.log
-  val libDir = (Universal/stagingDirectory).value / "lib"
-
-  // remove all `/module-info.class` from all jars
-  Files.walk(libDir.toPath)
-    .filter(_.toString.endsWith(".jar"))
-    .forEach { jar =>
-      val zipFs = FileSystems.newFileSystem(jar)
-      zipFs.getRootDirectories.forEach { zipRootDir =>
-        Files.list(zipRootDir).filter(_.toString == "/module-info.class").forEach { moduleInfoClass =>
-          logger.info(s"workaround for scala completion bug: deleting $moduleInfoClass from $jar")
-          Files.delete(moduleInfoClass)
-        }
-      }
-      zipFs.close()
-    }
+Universal/mappings ++= sbt.Path.directory(new File("joern-cli/src/main/resources/scripts"))
+
+// remove module-info.class from dependency jars - a hacky workaround for a scala3 compiler bug
+// see https://github.com/scala/scala3/issues/20421
+val moduleInfoLocation = "/module-info.class"
```

with the other changes, this then needs to be a plain file name without path (which also makes me more confident it won't break on Windows)

```suggestion
val moduleInfoLocation = "module-info.class"
```

---

this is the path in java's ZipFilesystem which is identical for windows.